docs/features/REST-unregisterhubsite-method.md

```diff
@@ -0,0 +1,56 @@
+---
+title: UnRegisterHubSite REST method
+description: UnRegisterHubSite REST method
+ms.date: 2/26/2018
+---
+
+# UnRegisterHubSite
+
+> [!IMPORTANT]
+> The hub sites feature is currently in preview and is subject to change. It is not currently supported for use in production environments.
+
+Unregisters a hub site so that it is no longer a hub site. It will become a regular site. Any sites associated with the hub site will no longer be associated. This can take up to an hour to propagate. If you want to speed up the process, first remove any associated sites before unregistering the hub site.
```

Comment:
I'm pretty sure the latter part of this paragraph is incorrect: when a hub site is removed, if I remember right, the associated sites will still have their hubSiteId set to the ID of the removed hub site (the change doesn't propagate). You should check with Dan to verify that.

Reply:
@ecraig12345 this was info I think Melissa relayed to me. We should probably discuss more.